src/ol/format/wmsgetfeatureinfo.js

```diff
@@ -64,6 +64,22 @@ ol.format.WMSGetFeatureInfo.layerIdentifier_ = '_layer';
 
 
 /**
+ * @return {Array.<string>} layers
+ */
+ol.format.WMSGetFeatureInfo.prototype.getlayers = function() {
```

Comment:
Shouldn't this be camel case (`getLayers`)?

Reply:
ouch. yes, changed